crossfire/clients/occurrences.py

```diff
@@ -164,3 +177,43 @@ def __call__(self):
             return GeoDataFrame(self.data)
 
         return self.data
+
+
+def _flatten_df(row, column_name):
+    column_data = row[column_name]
+    return Series(
+        {f"{column_name}_{key}": value for key, value in column_data.items()}
+    )
+
+
+def is_empty(data):
+    if HAS_PANDAS and isinstance(data, DataFrame):
+        return data.empty
+    return not data
+
+
+def flatten(data, nested_columns=None):
+    nested_columns = set(nested_columns or NESTED_COLUMNS)
+    if not nested_columns.issubset(NESTED_COLUMNS):
+        raise NestedColumnError(nested_columns)
+    if is_empty(data):
+        return data
+    if HAS_PANDAS and isinstance(data, DataFrame):
+        keys = set(data.columns) & nested_columns
+        for key in keys:
+            data = concat(
+                (
+                    data.drop(key, axis=1),
+                    data.apply(_flatten_df, args=(key,), axis=1),
+                ),
+                axis=1,
+            )
+
+        return data
+
+    keys = set(data[0].keys()) & nested_columns
+    for item in data:
+        for key in keys:
+            item.update({f"{key}_{k}": v for k, v in item.get(key).items()})
+            item.pop(key)
```

Comment:
What about extracting this as a `_flatten_list` to be a parallel of `_flatten_df`?

Reply:
@cuducos in the last commit I did this extraction. Hope the way I have done is fine.